translations/en.json

```diff
@@ -179,6 +179,11 @@
     "chat-settings": "Chat settings",
     "chat-with-friends": "Chat privately with friends",
     "chats": "Chats",
+    "check-before-syncing": "Check before syncing",
+    "check-before-syncing-doc-description":"To sync your devices successfully, make sure to check and complete these steps:",
+    "check-before-syncing-doc-checkbox-1":"Both devices are on the same network",
+    "check-before-syncing-doc-checkbox-2":"Make sure you are logged in on the other device",
+    "check-before-syncing-doc-checkbox-3":"Disable the firewall on your device",
```

Comment:
spaces are still missing at other occurrences.

Reply:
Uh sorry for that, I forgot that I added so many translations in this PR. I'll fix them in this PR.